### PR TITLE
Fix timestamp precision issue

### PR DESCRIPTION
**Pull request by snehilzs**
Made adjustments for timestamp precision
Removed all utc.now() values from code

### CODE REVIEW

File: v2/Cargo.toml [hunk 1]
```diff
@@ -27,7 +27,10 @@ chrono = { version = "0.4.31", features = ["serde"] }
 env_logger = { workspace = true }
 futures = "0.3.28"
 git-version = { workspace = true }
-influxdb2 = { version = "0.4.5", features = ["rustls"], default-features = false }
+humantime = "2.1.0"
```
Comment (JEnoch) [inline v2/Cargo.toml:30]: Chrono is already a dependency and can do RFC3339 formatting with nanoseconds precision.
Better to use it than to import an additional dependency for this.

File: v2/src/lib.rs [hunk 2]
```diff
@@ -832,51 +790,54 @@ fn key_exprs_to_influx_regex(path_exprs: &[&keyexpr]) -> String {
...
+    use humantime::format_rfc3339_nanos;
+    use std::fmt::Write;
+    match t {
         TimeExpr::Fixed(t) => {
-            let time_in_subsecs = t
-                .duration_since(UNIX_EPOCH)
-                .expect("Time went backwards")
-                .as_secs_f64();
-            Ok(time_in_subsecs)
+            let tm = t + Duration::from_millis(i * 10); //adding 10ms for timebound
```
Comment (JEnoch) [inline v2/src/lib.rs:833]: I think 10ms is already a too big gap.
What we want here is a distinction between inclusive or exclusive timestamp, for the case a timestamp in the database has the exact same value than a stop/start range value. Knowing that in Flux language, the `range.start` is always inclusive and the `range.stop` is always exclusive.

So I think 1ns is the most accurate distinction we can do to include or exclude a timestamp.
I.e. the `i: u64` parameter could be replaced with a `add_one_ns: bool` param

File: v2/src/lib.rs [hunk 3]
```diff
@@ -640,54 +639,13 @@ impl Storage for InfluxDbStorage {
...
-                    warn!(
-                        r#"Failed to decode zenoh Timestamp from Influx point with timestamp="{}": {:?}"#,
-                        zpoint.timestamp, e
-                    );
-                    continue;
-                }
-            };
-            let value = Value::new(payload).encoding(encoding);
-            result.push(StoredData { value, timestamp });
+        for i in &query_result {
```
Comment (JEnoch) [inline v2/src/lib.rs:642]: Why this change in conversion of `zpoint` to `StoredData` ?
Here the encoding will be lost, and the value always returned as a `String` (since `ZenohPoint.value` is a `String`). Moreover the case of a `base64` string is no longer treated.

**Review by JEnoch — CHANGES_REQUESTED** [review]

File: v2/Cargo.toml (hunk 1 above)
Comment (snehilzs) [reply v2/Cargo.toml]: Added the changes.

File: v2/src/lib.rs (hunk 2 above)
Comment (snehilzs) [reply v2/src/lib.rs]: Added the changes.

File: v2/src/lib.rs (hunk 3 above)
Comment (snehilzs) [reply v2/src/lib.rs]: Fixed.

**Review by JEnoch — APPROVED** [review]
